docker-compose.yml

```diff
@@ -70,3 +71,14 @@ services:
     networks:
       - dev
       - test
+    ports:
+      - 5432:5432
+
+  gcs:
```

Comment:
How does this image handle persistance? Allowing connectivity from both dev and tests networks might mess something up.

Reply:
it stores files in memory. i don't think we need other persistence here